ingestfile

```diff
@@ -39,13 +42,16 @@ _usage(){
 user_input="${*}"
 OPTIND=1
 
-while getopts ":F:pnhei" OPT ; do
+while getopts ":F:pnheiu:I:m:" OPT ; do
     case "${OPT}" in
         F) FORMULA="${OPTARG}";;
         e) EDIT_MODE="N" ;;
         p) PRESERVATION_MODE="Y" ; DELIVERYTOOMNEON="N" ; DELIVERACCESSCOPIES="N" ; DELIVERYTOAIPSTORAGE="N" ;;
         n) DELIVERYTOOMNEON="N" ; DELIVERACCESSCOPIES="N" ;;
         i) DELIVERYTOOMNEON="Y" ; DELIVERACCESSCOPIES="N" ; DELIVERYTOAIPSTORAGE="Y" ;;
+        u) OPERATOR="${OPTARG}" ;;
```

Comment:
any reason to add another variable for the operator, rather than re-use the $OP used elsewhere?